tests/Bundle/BundleMetadataTest.php

```diff
@@ -103,6 +108,7 @@ public function testBundleName()
         $bundleMetadata = new BundleMetadata($bundle, [
             'application_dir' => 'Application',
             'namespace' => 'Application',
+            'namespace_prefix' => '',
```

Comment:
Can we add one test that defines this parameter with a value?

Reply:
this is now down in `testWithNamespacePrefix`